src/java/org/apache/cassandra/service/StorageService.java

```diff
@@ -3849,6 +3849,12 @@ public int forceKeyspaceCleanup(int jobs, String keyspaceName, String... tables)
         if (SchemaConstants.isLocalSystemKeyspace(keyspaceName))
             throw new RuntimeException("Cleanup of the system keyspace is neither necessary nor wise");
 
+        InetAddressAndPort localAddress = FBUtilities.getBroadcastAddressAndPort();
+        if (tokenMetadata.getPendingRanges(keyspaceName, localAddress).size() > 0)
+        {
```

Comment:
nit: if `if` contains just one statement, we tend to omit the braces to save some space.

nit: `FBUtilities.getBroadcastAddressAndPort()` can be used directly in `if`, I would statically import that method as well.